scenarios/16_Choosing_the_Best.cfg

```diff
@@ -76,14 +77,25 @@
         [part]
             story=_"Who was she? She wore a dancer's mask, but somehow I knew she wasn't one of dancers from a few hours earlier. There was something familiar in the way she moved, breathed and gasped. Maybe if I had been sober, I would have recognised her even with the mask. But I wasn't, and I didn't care."
 #ifdef __UNUSED__
-            background=story_images/story_maska.png
+            background=story_images/story_maska.png # still exists in ANO_art repo; removed due to quality
 #endif
         [/part]
         [part]
             story=_"She left as quietly as she came, as if she were just a dream. But I still could smell her in the air, and my hands still remembered her as well. Whoever she was, I was certain I would recognise her, if I could only touch her again."
         [/part]
         [part]
-            story=_"But I had no such chance. The next day, Gwidle woke me before dawn and I came to my soldiers, to call the twelve best. Most of them were in better shape than I, at least having had some sleep that night."
+#ifdef EASY
+            #po: EASY difficulty:
+            story=_"But I had no such chance. The next day, Gwidle woke me before dawn and I came to my soldiers, to call the eighteen best. Most of them were in better shape than I, at least having had some sleep that night."
```

Comment:
I wonder if you can make the number a variable so this isn't split into multiple strings? Not really a big deal but would be nice. Here and below, also in S17.

Reply:
The issue is that it'd have to be two separate variables, one as a number, and another with the number written out textually as a string. Also, string concatenation can make things more difficult for translators...

Reply:
Perhaps nemaara was suggesting that the number doesn't need to be formatted as a word? I think that's probably the only viable way to turn this into a variable, anyway.

In theory, a generic way of translating a number to a localized phrase can probably be made, but I'm not sure what kinds of roadblocks it would run into, and even if it's viable, you'd probably want to integrate it into the Wesnoth core in order for it to be of general use.

Reply:
Yeah exactly it doesn't need to be a word. But fine, you can leave it.